src/fonts/inter.scss

```diff
@@ -82,6 +82,33 @@ Usage:
   src: url('woff2/Inter-roman.var.woff2') format('woff2');
 }
 
+/* Minimal font file with limited glyphs and layout features (only kern).
+   Should be enough for most en-US pages. This was generated by looking at
+   unicode ranges needed to display most of our en-US pages with glyphhanger
+   and manual tweaks to add several characters that are likely to appear.
+
+   In serverHtml component, we preload subsets corresponding to the locale,
+   so a request for /en-US/something would preload this font because of its
+   name.
+
+   The corresponding command to generate the font is:
+   pyftsubset Inter-roman.var.woff2
+    --output-file=Inter-roman-minimal-subset-en-US.var.woff2
+    --flavor=woff2
+    --layout-features=kern
+    --no-hinting
+    --unicodes=<unicode range>
+*/
+@font-face {
```

Comment:
I think it would be useful to add a comment explaining that this `@font-face` definition needs to be the last in file order because of how unicode range works.

Reply:
Good point. I'll do that in my follow-up because the comment will make more sense (and be even more important) once I have several subsets that overlap each other.